modules/ban_enforce.py

```diff
@@ -15,13 +17,19 @@ def on_mode(self, event):
                     bans.append(arg)
 
             if bans:
+                affected = 0
```

Comment:
can you move this and the `realmax` code above `if bans:`? the below if statement (`if kicks:`) won't evaluate true if this one doesn't evaluate true, which means this code will always work fine, but defining something in an if statement scope to then be used in a different if statement scope that is a sibling scope seems wrong